packages/gr-air-modes/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=gr-air-modes
+pkgver=396.0b6c383
+pkgrel=1
+pkgdesc="Gnuradio tools for receiving Mode S transponder signals, including ADS-B."
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
+groups=('blackarch' 'blackarch-radio')
+url='https://github.com/bistromath/gr-air-modes'
+license=('GPL3')
+depends=('gnuradio' 'gnuradio-companion' 'gnuradio-osmosdr' 'python2-numpy'
+         'python2-scipy' 'python2-pyzmq' 'sqlite' 'libuhd')
+makedepends=('cmake' 'git')
+source=("git+http://github.com/bistromath/gr-air-modes.git"
+        "modes_gui.patch")
+sha512sums=('SKIP'
+            '0b2149ff9b5b6e198380a01018eb385162493ec6ab813d4c83c46f19987a146a0a30fcc9f60c7c8c68b0ac0c18e5b5b08be9bf11000779d008d2483990cbab52')
+
+
+pkgver() {
+  cd "$pkgname"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
There is a tag available. Could you add it here @noptrix? Thanks.

Reply:
hello @FFY00 i have test to build the package with the tag, but that's not compatible with the current version of gnu-radio

Reply:
What do you mean? I am just saying add the tag too to `pkgver`.